### ISSUE TITLE
Vertical resource view - Minimum width

### ISSUE THREAD

**Issue by alan1o** [fullcalendar/fullcalendar-scheduler#110]
If dealing with lots of resources, currently each resource column just gets more and more compressed. It would be good if we could set a minimum width and have a horizontal scroll bar appear for resources that are unable to fit within the existing view.

**Comment by arshaw** [comment]
i was considering this, bug decided against it because `timeline` view is much better at displaying a lot of resources. i'll leave this open for discussion though.

**Comment by vaidast** [comment]
That option would be really nice. Assuming that some users of the calendar who have one/two resources would rather use vertical view (much easier for an eye, something like google calendar). Then assume that resources grow a little to 5-8, so user would need to switch to timeline, if user is used to vertical view it's better to maintain vertical view. This option would be very useful for mobile device users. If you have more than 3-4 resources, mobile screens suppress columns and resource titles overlap with other resource titles. That vertical resource view becomes useless on a mobile phone with only 3-4 resources. I am all for it.

**Comment by arshaw** [comment]
ticket in core: https://github.com/fullcalendar/fullcalendar/issues/3022

**Comment by avanti-technologies** [comment]
Looking forward to this.

**Comment by henritoivar** [comment]
Any ETA on this?

**Comment by curreli** [comment]
+1, any ETA?

**Comment by Cici2014** [comment]
+1, need such feature too.

**Comment by cormip** [comment]
+1

**Comment by alan1o** [comment]
Any chance of this in next version @arshaw ?

**Comment by espen** [comment]
@alan1o "Comments such as +1 or “when will this be ready” will not only receive a penalty in the ranking system, they will spam other watchers of the thread " See https://fullcalendar.io/blog/2016/07/renovated-issue-tracker/

**Comment by gflauder** [comment]
I would agree with Vaidast's comment on the 13th. The Vertical view is fantastic especially in 3 day mode for scheduling. Easy for a user to see and navigate scheduling within the day or three. I personally think this view is much easier to visualize than a horizontal timeline. Would be a welcomed feature from my perspective.

**Comment by ankitpise** [comment]
+1

**Comment by pareshhp** [comment]
Try using following css , it makes looks better.

.fc-view-container { overflow-x: scroll; } 
.fc-view.fc-agendaDay-view.fc-agenda-view{ width: 500%; }
.fc-view.fc-agendaTwoDay-view.fc-agenda-view{  width: 500%; }

**Comment by sp00m** [comment]
I would be happy to help bringing this feature in, as we need it as well. Any clue from where to start on my local fork? Have you thought about the way this could be implemented already? What about trying the accepted answer on https://stackoverflow.com/q/1312236/1225328, i.e. having the first column set to `position: absolute`?

**Comment by sp00m** [comment]
Based on @pareshhp idea, I've setup a ~~nice robust solution~~ hacky temporary workaround to have an horizontal scrollbar: http://plnkr.co/edit/MGbUiMczvrkV32dWW0gb.

**Comment by richlandhosting** [comment]
.fc-view-container { overflow-x: scroll !important; }
.fc-ltr .fc-axis { position:relative; }

.fc-view.fc-agendaDay-view.fc-agenda-view - I dynamically adjust width based on how many resources are shown.

Above should allow many resources in agenda view with horizontal scrolling and keeps the times on the left as you scroll horizontal.

**Comment by vishaldudhatra** [comment]
+1

I have same issue with horizontal scrolling in Agenda views, please see in below screenshot.
[https://www.screencast.com/t/nXszWpNxi7](url)
I was trying to resolve this issue using css property, but its not proper work like vertical scroll.

**Comment by dennyak47** [comment]
I believe that horizontal scroll should be supported, because we need to make the agenda function better. As for whether it is convenient for user, we let them to decide.
